getting-started/README.md

```diff
@@ -1,33 +1,35 @@
-# Getting Started
+# **Getting Started**
 
-## Course Overview
+# Course Overview
 
 Welcome to Data Stack Academy's Data Engineering Bootcamp. By the end of this course you will be a well-seasoned Cloud Engineer. Throughout the course you will learn to use nearly all of Google Cloud Platform (GCP) analytical services. This includes services such as Google BigQuery, Dataflow, Dataproc, Cloud Pub/Sub, Cloud Run, Cloud AI, BigTable, and Cloud Composer.
 
-We have specifically chosen Google Cloud over other cloud platforms because we believe GCP to offer the best combination of services in the data enginering space, and because the GCP services are the closest to their open source Big Data counterparts, such as Apache Hadoop, Hive, Hbase, Spark, Beam, Kafka, Airflow, and more. Learning the material in this course will also prepare you to become a professional Big Data developer.
+We have specifically chosen Google Cloud over other cloud platforms because we believe GCP to offer the best combination of services in the data engineering space, and because the GCP services are the closest to their open source Big Data counterparts, such as Apache Hadoop, Hive, Hbase, Spark, Beam, Kafka, Airflow, and more. Learning the material in this course will also prepare you to become a professional Big Data developer.
 
 We believe that we have put together a course curriculum that is second to none. Learning these data and cloud skills will enable you to enter the job market as a modern Data Engineer. These days, nearly all startup companies and applications run on cloud infrastructure and seek to maximize their competitive advantage by leveraging large amounts of data.
 
 We hope you enjoy this course, and are confident that it will teach you all the necessary skills to start a rewarding career in Data Engineering.
 
-## Setting up your development environment
+# Setting up your development environment
 
 Before we begin to go through the course material, it is important for everyone to set up their development environment so that you can easily access all the resources we will use for this project (code, repository material, cloud resources). Here is an overview of the setup steps:
```

Comment:
```suggestion
Before we begin to go through the course material, it is important for everyone to set up their development environment so that you can easily access all the resources we will use for this project (code, repository material, cloud resources).
Here is an overview of the setup steps:
```